AppBuilder/platform/FilterComplex.js

```diff
@@ -148,6 +154,14 @@ function _toExternal(cond, fields = []) {
    }
 }
 
+function _uiQueryOptionId(fieldId) {
+   return `byQueryField-query-option-${fieldId}`;
```

Comment:
Question:  suppose I create a condition like:

FieldA  in_query_field  QueryA
or
FieldA in_query_field  QueryB

I'm reusing `FieldA` in both condition statements.  How can we uniquely identify both of those values if our `_uiQueryOptionId()` is reusing the same `fieldId` for each?

Reply:
@Hiro-Nakamura 
This function returns a reference to the input for our custom AB options. 
If you will notice `webix.query` method always handles only one input at a time.

When we add or edit a filter, it hides the other filters by removing their HTML DOM elements.

Otherwise, Having multiple filters with the same field ID ensures that the reference can get our custom value each time.

Reply:
OK, if it only opens one input at a time, this should work pretty well then.